src/sssom/util.py

```diff
@@ -1105,62 +1105,25 @@ def reconcile_prefix_and_data(
     """
     # Discussion about this found here:
     # https://github.com/mapping-commons/sssom-py/issues/216#issue-1171701052
+    converter = msdf.converter
+    converter = curies.remap_curie_prefixes(converter, prefix_reconciliation["prefix_synonyms"])
+    converter = curies.rewire(converter, prefix_reconciliation["prefix_expansion_reconciliation"])
```

Comment:
Use constants instead of strings to satisfy SRP?

Reply:
I don't know what SRP means, but let's save updating this data structure for a subsequent PR

Reply:
[Single responsibility principle](https://en.wikipedia.org/wiki/Single-responsibility_principle): classes/methods/functions should only have one reason to change. If this string changes for whatever reason, many different classes and places in the code will have to change as well.